### PR TITLE
Fixed empty dropdown showing when no results are found

### PR DESCRIPTION
**Pull request by blackbaud-GavinNicol**
Addresses issue: #43

### CODE REVIEW

File: src/app/public/modules/autocomplete/autocomplete.component.html [hunk 1]
```diff
@@ -26,6 +26,11 @@
           </button>
 
         </sky-dropdown-item>
+        <sky-dropdown-item *ngIf="searchResults.length === 0">
+          <div class="sky-autocomplete-no-results-found">
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/autocomplete/autocomplete.component.html]: We have a few CSS classes we can use for the padding. Mind replacing the `sky-autocomplete-no-results-found` class with `sky-padding-squish-default`? You can also remove the CSS styles from the stylesheet.

**Comment by blackbaud-GavinNicol** [comment]
Good catches, both of those issues should be addressed now

File: src/app/public/modules/autocomplete/autocomplete.component.html [hunk 2]
```diff
@@ -26,6 +26,11 @@
           </button>
 
         </sky-dropdown-item>
+        <sky-dropdown-item *ngIf="searchResults.length === 0">
+          <div class="sky-padding-squish-default">
```
Comment (Blackbaud-ToddRoberts) [inline src/app/public/modules/autocomplete/autocomplete.component.html]: the "not found" text should use sky-deemphasized class

Comment (blackbaud-GavinNicol) [reply src/app/public/modules/autocomplete/autocomplete.component.html]: 👍 Updated!

**Pull request merged**